README.md

```diff
@@ -38,3 +38,17 @@ To use gpu encoding, use a VAAPI codec (for ex. `h264_vaapi`) and specify a GPU
 ```
 wf-recorder -f test-vaapi.mkv -c h264_vaapi -d /dev/dri/renderD128
 ```
+
+## with [Sway](https://swaywm.org/)
+
+In your Sway config file, add those key bindings to start and stop wf-recorder :
+
+```
+bindsym Ctrl+Print exec wf-recorder -f ~/recording_$(date +"%Y-%m-%d_%H:%M:%S.mp4")
+bindsym Ctrl+Shift+Print exec wf-recorder -g "$$(slurp)" -f ~/recording_$(date +"%Y-%m-%d_%H:%M:%S.mp4")
```

Comment:
Is this double $$ a typo in `-g "$$(slurp)"`?

Reply:
no it's for escaping the `$`, otherwise it's considered a configuration variable